### DISCUSSION TITLE
Enable exports for specific albums only

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#593] [Enhancements]
Currently while exporting data, we perform a bulk export for all the albums.

Not being able to opt out of certain albums could result in undesired duplicates, in case the source of an album is already present on the user's machine.

Users should have an option to choose the albums they wish to export, within the export screen.

**Comment by ua741** [comment]
Using CLI, you can perform export of [selected albums](https://github.com/ente-io/ente/blob/main/cli/docs/generated/ente_export.md).

**Reply by Trekky12** [reply]
Not all tech-savy users would like to configurate the CLI, so this would be a big improvement for the desktop version for them, especially when the disk storage is limited and not all albums are needed on the local disk.

**Reply by jpstroop** [reply]
Is it just me, or does the album-level export not include the JSON sidecar file with location data?
